test/recipes/81-test_cmp_cli.t

```diff
@@ -200,7 +200,7 @@ sub test_cmp_cli_aspect {
             }
         }
     };
-    # not unlinking test.certout_*.pem, test.cacerts.pem, and test.extracerts.pem
+    # not unlinking test.certout*.pem, test.cacerts.pem, and test.extracerts.pem
 }
 
 indir data_dir() => sub {
```

Comment:
I would start by replacing this line with `subtest "name-of-subtest" => sub {`

Yes, it probably means that you will have to use `data_dir` and `data_file` to reach the source input files where you hadn't expected to have to.  However, you don't need to do much with your output files, just write them in the location this test runs in.

Reply:
The CMP CLI-based tests use much greater variety of input files than output files.
Therefore it was less effort to prepend the output files with BLDTOP than it would be the other way round.
And a stronger point against reverting this to prepending the input files is that they dynamically depend on the test server used (the Mock server is just one of them; at Siemens we use a couple of others, too) while on the other hand the main test configuration file `test.cnf`, which references several server-dependent input files by relative file names, is static.

Reply:
In that case, I would like to see a comment to that effect, as this isn't "normal" behaviour for our test recipes.

Reply:
I've added a comment giving essentially the explanation that I wrote above.